internal/userutils/userutils_bwrap_test.go

```diff
@@ -85,6 +85,26 @@ testgroup:x:1001:testuser`
 	require.Equal(t, groupContents, output)
 }
 
+func TestLockAndLockAgainGroupFile(t *testing.T) {
+	require.Zero(t, os.Geteuid(), "Not root")
+
+	err := userutils.LockGroupFile()
+	require.NoError(t, err, "Locking once it is allowed")
+
+	err = userutils.LockGroupFile()
+	require.Error(t, err, "Locking again should not be allowed")
+
+	err = userutils.UnlockGroupFile()
+	require.NoError(t, err, "Unlocking should be allowed")
+}
```

Comment:
This is already tested in `TestLockAndUnlockGroupFile`: https://github.com/ubuntu/authd/blob/92b0d73c99fad6cd7f4f1bb76a41ad356d573858/internal/userutils/userutils_bwrap_test.go#L44-L47

Reply:
Yeah, I will remove it from that one since that's a too long test doing too many checks already, while this is more concise.